friends/src/App.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+import FriendsList from './components/FriendsList';
+import './App.css';
+
+class App extends Component {
+	constructor() {
+		super();
+		this.state = {
+			data: [],
+			name: '',
+			age: '',
+			email: '',
+		};
+	}
+	componentDidMount() {
+		axios.get('http://localhost:5000/friends')
+      .then( response => this.setState({ data: response.data }))
+      .catch(err => console.log(err))
+  }
+	
+
+  inputChangeHandler = e => {
+    this.setState({ [e.target.name]: e.target.value });
```

Comment:
This is a great pattern for handling multiple inputs as it allows us to write less code and handle changing inputs with a single handler 👍